Functions/Mock.Tests.ps1

```diff
@@ -2216,11 +2216,92 @@ Describe "Mock definition output" {
     }
 }
 
-Describe 'Mocking using ParameterFilter with scriptblock' {
-    $filter = [scriptblock]::Create( ('$Path -eq ''C:\Windows''') )
-    Mock -CommandName 'Test-Path' -ParameterFilter $filter
+Describe 'Mocking using ParameterFilter' {
```

Comment:
@clcaldwell Some of these tests seem to be very similar. They could be candidate to be refactored using the **TestCases** option